src/MoBi.Assets/AppConstants.cs

```diff
@@ -1092,10 +1092,12 @@ public static class MenuNames
          public static readonly string AddBuildingBlocks = "Add Building Blocks...";
          public static readonly string LoadBuildingBlocks = "Load Building Blocks...";
          public static readonly string LoadBuildingBlocksFromTemplate = "Load Building Blocks from Template...";
-         public static readonly string MakeSettingsProjectDefault = "Make Settings Project Default";
-         public static readonly string RefreshSettingsFromProjectDefault = "Refresh Settings from Project Default";
+         public static readonly string CommitToProjectDefaults = "Commit to project defaults";
```

Comment:
Commit seems to be a bit technical. Is this what we are using in PKsim?

Reply:
I wasn't aware you could make settings in a simulation become the project default.

We used to use the word 'make' instead of 'commit'. That's probably better. This is somewhat different from `Commit to building blocks` .